testsuite/features/step_definitions/command_steps.rb

```diff
@@ -156,6 +156,18 @@
   end
 end
 
+Then(/^file "([^"]*)" should contain "([^"]*)" on server$/) do |file, content|
+  output = sshcmd("grep -F '#{content}' #{file}", ignore_err: true)
+  raise "'#{content}' not found in file #{file}" if output[:stdout] !~ /#{content}/
+  "\n-----\n#{output[:stderr]}\n-----\n"
```

Comment:
I don't understand the last line in those two methods.
Steps definitions don't have a return value, we why need them?

Reply:
Removed line.